ci/prow/make_config.go

```diff
@@ -1214,7 +1214,8 @@ func generateTestGroup(projName string, repoName string, jobNames []string) {
 			extras["short_text_metric"] = "coverage"
 			// Do not alert on coverage failures (i.e., coverage below threshold)
 			extras["num_failures_to_alert"] = "9999"
-		case "istio-1.0.7-mesh", "istio-1.0.7-no-mesh", "istio-1.1.7-mesh", "istio-1.1.7-no-mesh":
+		case "istio-1.0.7-mesh", "istio-1.0.7-no-mesh", "istio-1.1.7-mesh", "istio-1.1.7-no-mesh",
```

Comment:
This is getting worse, we should definitely address this sooner than later. :)

Reply:
Totally agree. As we discussed earlier, we can reuse the data already parsed while generating Prow config to generate testgrid config, which requires non trivial amount of refactoring

Reply:
Indeed.